indexer/src/main.rs

```diff
@@ -58,6 +57,8 @@ pub enum SinkConfig {
     Cardano { db: DbConfig, network: String },
 }
 
+pub enum Network {}
```

Comment:
it's not used anywhere, remove probably?

Reply:
I was going to replace the strings ("mainnet", "preprod") in `process_genesis`, by using serde's `#[serde(rename_all = "lowercase")]` 

But then changed my mind since it's a bit out of scope and forgot to remove this.